src/Resources/config/services.yml

```diff
@@ -52,14 +52,14 @@ services:
             - { name: kernel.event_listener, event: welp.mailchimp.delete, method: onDelete }
 
     # Commands
-    Welp\MailchimpBundle\Command\SynchronizeMergeFieldsCommand:
+    welp_mailchimp.command.synchronize_merge_fields:
```

Comment:
If you wanbt to use id, you have to set class in the definition. 
For example:
```
welp_mailchimp.command.synchronize_subscribers:
  class: Welp\MailchimpBundle\Command\SynchronizeSubscribersCommand
  tags:
     - { name: 'console.command', command: 'welp:mailchimp:synchronize-subscribers' }   
```

Reply:
Yes obviously! my bad...